esx_service/vmdk_ops.py

```diff
@@ -1636,7 +1636,14 @@ def main():
 
         kv.init()
         connectLocalSi()
+        err_msg = vm_listener.start_vm_changelistener()
```

Comment:
Hostd (and `si`) may not not be available after ESX reboot. See #976 This means VMListener won't start after ESX reboot.